tests/tests/test_multi_tenancy.py

```diff
@@ -89,19 +89,17 @@ def set_correct_tenant_token(token):
     @pytest.mark.usefixtures("multitenancy_setup_without_client")
     def test_artifacts_exclusive_to_user(self):
         users = [
-            {"email": "greg1@greg1.com", "password": "hunter2hunter2", "username": "greg1"},
-            {"email": "greg2@greg2.com", "password": "hunter2hunter2", "username": "greg2"},
-            {"email": "greg3@greg2.com", "password": "hunter2hunter2", "username": "greg3"}
+            {"email": "foo1@foo1.com", "password": "hunter2"*2, "username": "foo1"},
+            {"email": "bar2@bar2.com", "password": "hunter2"*2, "username": "bar2"},
```

Comment:
The `*2` syntax is just weird... Isn't this harder to read than what was already there?